auth/server/src/main/java/com/seagates3/util/BinaryUtil.java

```diff
@@ -239,8 +235,7 @@ public static String base64EncodedHash(String text) {
 
             return md.digest();
         } catch (NoSuchAlgorithmException ex) {
-            IEMUtil.log(IEMUtil.Level.ERROR, IEMUtil.SHA256_UNAVAILABLE,
-                    "Algorithm SHA-256 not available", null);
+            LOGGER.error"Algorithm SHA-256 not available.");
```

Comment:
starting bracket is missing ..pls add it.
After all these fixes please try to build auth server  as ,
cd cortx-s3server/auth/
./mvnbuild.sh clean
./mvnbuild.sh package

Reply:
added '(' which was missing.